extensions/amp-animation/0.1/web-animations.js

```diff
@@ -209,12 +209,44 @@ export class AnimationWorkletRunner extends AnimationRunner {
 
     return `
     registerAnimator('anim${++animIdCounter}', class {
+      constructor(options = {
```

Comment:
could you add a TODO or issue to move this code to its own file so that it gets linted, etc... may need to change the build process a bit, but I think amp-bind does this for running code in a webworker.

Reply:
Yup I just had this discussion with @kristoferbaxter - tl;dr - would need a change in the build process.